### PR TITLE
Mark all London EIPs as Last Call

### PR DESCRIPTION
**Pull request by axic**
Suggested on ACD by @timbeiko, with the end-date set to the date of mainnet fork.

### CODE REVIEW

**Review by vbuterin — APPROVED** [review]

**Review by MadeofTin — APPROVED** [review]

File: EIPS/eip-3529.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ eip: 3529
 title: Reduction in refunds
 author: Vitalik Buterin (@vbuterin), Martin Swende (@holiman)
 discussions-to: https://ethereum-magicians.org/t/eip-3529-reduction-in-refunds-alternative-to-eip-3298-and-3403-that-better-preserves-existing-clearing-incentives/6097
-status: Draft
```
Comment (axic) [inline EIPS/eip-3529.md:6]: I think technically this should have gone through the `Review` state first, but it was extensively discussed during 2 (or 3) ACD meetings.

@timbeiko potentially the rule of considering only EIPs in the `Review` )or `Last Call`) state for final inclusion decision could be more strictly enforced.

**Comment by axic** [comment]
Apparently the new automerge bot can't handle multiple files:
```
sorry only 1 file is supported right now
[
  'An Exception Occured While Linting: sorry only 1 file is supported right now'
]
```

File: EIPS/eip-3529.md (hunk 1 above)
Comment (MadeofTin) [reply EIPS/eip-3529.md:6]: It Technically okay to have skipped the step, but it shouldn't be the normal case. I remember writing something about what steps are allowed to be skipped, but I don't know it if ever got merged anywhere.

Being better with EIP-1 would help enforce some of these things as of now, what is and what isn't a rule isn't always clear.

**Comment by MadeofTin** [comment]
Probably worth moving through this one and getting someone to hit the merge when we get the checks. Can we put in a feature request for multiple files? 😅

**Comment by MadeofTin** [comment]
Actually he have all the checks we need as Vitalik is author on 3 of them. 👏

**Comment by axic** [comment]
I tried triggering the old automerge bot, but the reason that won't work because it wants a clean CI state, i.e. no failures. Since the new bot displays as a failure, the old bot won't do anything. (It has been rendered useless.)

File: EIPS/eip-3529.md (hunk 1 above)
Comment (timbeiko) [reply EIPS/eip-3529.md:6]: @axic fair enough. I think it's more important to move them to `Last Call` now though given that's what triggers folks to look at the EIPs, so I would rather go Draft -> Last Call directly vs. delaying it by a few days/weeks.

**Comment by MicahZoltu** [comment]
I don't think the old bot handled this situation either.

**Comment by axic** [comment]
> I don't think the old bot handled this situation either.

It definitely worked for merging multiple files, but you are correct that since there was a status change it would not have merged it.

**Pull request merged**